css/style.css

```diff
@@ -83,11 +112,26 @@ ul.stacked-list {
   color: var(--light-red);
 }
 
+#image .image-responsive img,
+#card img {
+  border-radius: 0%;
+}
+
 .modal-container {
   background-color: var(--light-gray);
   padding: 1rem;
 }
 
+.modal div,
+.modal div button,
+.stacked-list span,
+.form-container div,
+.form-container label,
+.input-container label,
+.input-container span {
```

Comment:
Instead of having multiple classes there you can create a utility class by the name of color-black and use it in these divisions

Reply:
ok will try if classes are not needed, Thanks